addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/PresenceDetection.java

```diff
@@ -128,7 +132,18 @@ public void setPingMethod(PingMethod pingMethod) {
         this.pingMethod = pingMethod;
     }
 
+    /**
+     * Enables or disables ARP pings. Will be automatically disabled if the destination
+     * is not an IPv4 address. If the feature test for the native arping utility fails,
+     * it will be disabled as well.
+     * 
+     * @param useARPping Enable or disable ARP ping
+     * @param arpPingUtilPath The file path to the utility
+     */
     public void setUseARPping(boolean useARPping, String arpPingUtilPath) {
```

Comment:
Minor: I did not notice previous time, but this should be either `useARPPing` or the more readable `useArpPing`